adapter/internal/messaging/connection.go

```diff
@@ -110,19 +111,52 @@ func connectionRetry(key string) (*Consumer, *amqp.Connection, error) {
 
 			if key != "" && len(key) > 0 {
 				logger.LoggerMsg.Infof("Retry attempt %d for the %s to connect with topic %s has failed. Retrying after %d seconds", i,
-					amqpURIArray[j].url, key, amqpURIArray[j].connectionDelay)
+					maskURL(amqpURIArray[j].url), key, amqpURIArray[j].connectionDelay)
 			} else {
-				logger.LoggerMsg.Infof("Retry attempt %d for the %s has failed. Retrying after %d seconds", i, amqpURIArray[j].url, amqpURIArray[j].connectionDelay)
+				logger.LoggerMsg.Infof("Retry attempt %d for the %s has failed. Retrying after %d seconds", i, maskURL(amqpURIArray[j].url), amqpURIArray[j].connectionDelay)
 			}
 			time.Sleep(retryInterval)
 		}
-		if i == maxAttempt {
-			logger.LoggerMsg.Infof("Exceeds maximum connection retry attempts %d for %s", maxAttempt, amqpURIArray[j].url)
+		if i >= maxAttempt {
+			logger.LoggerMsg.Infof("Exceeds maximum connection retry attempts %d for %s", maxAttempt, maskURL(amqpURIArray[j].url))
```

Comment:
Lets change the message that Retries with increasing time will start

Reply:
A log is adde in L129

logger.LoggerMsg.Infof("Trying to connect exponentially for %s", maskURL(url)). 
Would this be ok?